### DISCUSSION TITLE
Spawner object will be copied after clicking！

### DISCUSSION THREAD

**Discussion by fenrir-liutiecheng** [mozilla/hubs#3818] [Hubs Cloud Questions] [answered]
Spawner object will be copied after clicking in the space, is there a way to prevent the object from being copied？

**Comment by pattersonbl2** [comment]
are you still having issues with this.

**Comment by pattersonbl2** [comment, marked as answer]
This looks to be a stale question I will mark as answered please unmark if issue still exist
